interface/reports/pat_ledger.php

```diff
@@ -590,24 +589,24 @@ function sel_patient() {
                                     </table>
                                 </div>
                             </td>
-                            <td class="align-middle h-100" align='left'>
-                                <table class="w-100 h-100" style='border-left: 1px solid;'>
+                            <td class="align-middle h-100 text-left">
+                                <table class="w-100 h-100 border-left" >
```

Comment:
If this can be converted to use the bs4 table class and table-responsive that would be great yet that might be too much right here